.github/workflows/ci-workflow.yml

```diff
@@ -53,10 +54,30 @@ jobs:
           name: boilerplate-app-nanoX-debug
           path: bin
 
+  job_nanoSP_build_debug:
```

Comment:
I suggest using a matrix strategy to reduce code duplication. A simple use is done in the review for dynamic exchange app build

Reply:
Nice, let's see.